### PR TITLE
add  returned from nested to nested spec

### PR DESCRIPTION
**Pull request by KamilMilewski**

### CODE REVIEW

File: test/docs/nested_test.rb [hunk 1]
```diff
@@ -131,6 +131,36 @@ class JsonValidate < Validate
...
+        step Nested(:compute_nested)
+        step :save
+        #~meths
+        include T.def_steps(:create, :save)
+        #~meths end
+      end
+      #:nested-with-pass-fast end
+
+      # pass fast
+      create.(seq: []).inspect(:seq).must_equal %{<Result:true [[:create, :just_pass_fast, :save]] >}
```
Comment (seuros) [inline test/docs/nested_test.rb]: why you assign it to a variable instead of using the Class directly ?

Comment (KamilMilewski) [reply test/docs/nested_test.rb]: got "inspired" by other tests above, I can use operation class directly here I guess

Comment (seuros) [reply test/docs/nested_test.rb]: It fine. ship it

Comment (KamilMilewski) [reply test/docs/nested_test.rb]: nice :+1:

**Pull request merged**